src/PyCall.jl

```diff
@@ -622,6 +622,9 @@ function anaconda_conda()
     if conda || !occursin("conda", unsafe_string(ccall(@pysym(:Py_GetVersion), Ptr{UInt8}, ())))
         return ""
     end
+    if isempty(pyprogramname)
+        return ""
+    end
```

Comment:
This means it will never find `conda` for statically linked libpython.  Is that what we want?

Reply:
Yes. But, I think that's fine because it's called from a Python executable. I don't think Python packages don't usually install Python package dependencies automatically at runtime.

Having said that, `pyimport("sys")[:executable]` is *technically* a valid option here.  But I don't think that's the right thing to do.  Please see #560